cmd/root.go

```diff
@@ -0,0 +1,143 @@
+package shanty
+
+import (
+	"net"
+	"os"
+	"time"
+
+	"github.com/prairir/Buoy/pkg/config"
+	"github.com/prairir/Buoy/pkg/tun"
+	"github.com/rs/zerolog"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/spf13/viper"
+)
+
+var cfgFile string
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:   "Buoy",
+	Short: "A mesh VPN",
+	Long: `A longer description that spans multiple lines and likely contains
+examples and usage of using your application. For example:
+
+Cobra is a CLI library for Go that empowers applications.
+This application is a tool to generate the needed files
+to quickly create a Cobra application.`,
+	// Uncomment the following line if your bare application
+	// has an action associated with it:
+	Run: Root,
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	cobra.CheckErr(rootCmd.Execute())
+}
+
+func init() {
```

Comment:
I remember you telling me that this is potentally an anti-pattern, could comment about it just in case a better solution is thought of.

Reply:
It is an anti pattern but this code is generated for us by `cobra init` and so im not worried about it